### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by gitressa**
Adds comment about Composer 2 probably not needing the plug-in.

### CODE REVIEW

**Review by zaporylie — CHANGES_REQUESTED** [review]
Thanks for your contribution. This can go in once #17 is in but could you rephrase it a bit? I think the key message we wanna send to people is that plugin is disabled if Composer 2 is detected and if everyone involved in project development is using Composer 2 then this plugin becomes redundant and can as well be just removed from the list of project dependencies.

**Comment by gitressa** [comment]
Sure, I have tried rephrasing the text, to make it clearer, and gave it a sub-heading to make it stand out a bit. What do you think?

**Review by zaporylie — APPROVED** [review]

**Comment by zaporylie** [comment]
I like it!

**Comment by gitressa** [comment]
Awesome! And thanks for making the plug-in the first place, and speeding up Composer and Drupal :)

**Comment by gitressa** [comment]
I accidentally closed and quickly reopened the PR, which might have made the continuous-integration/travis-ci/pr build fail ... Perhaps you can [trigger a re-build](https://stackoverflow.com/questions/17606874/trigger-a-travis-ci-rebuild-without-pushing-a-commit) somehow?

**Comment by zaporylie** [comment]
Build failure is expected and will be fixed by #17. Stay tuned.

**Comment by gitressa** [comment]
I see, thanks for clarifying. I tried yesterday to link this issue to the main issue, but without luck ... you probably need write permissions or something. From #17:

linked-issues
